alexthompson207/whats-cookin
# Feature/main page display

## Is this a fix or a feature?
- feature

## What is the change?
- HTML markup was built for the main page display
- CSS styling was added for design and responsive design
## What does it fix?
- N/A
## Where should the reviewer start?
- index.html
- style.css
## How should this be tested?
- open index.html from root directory
- page should have a title navigation area, search area, filter area, and list of recipes
- View the page at a width of 480px or less and the page should be responsive
## Screenshots (if appropriate):
